liblangutil/UniqueErrorReporter.h

```diff
@@ -58,11 +58,7 @@ class UniqueErrorReporter
 
 	void warning(ErrorId _error, std::string const& _description)
 	{
-		if (!seen(_error, {}, _description))
-		{
-			m_errorReporter.warning(_error, _description);
-			markAsSeen(_error, {}, _description);
-		}
+		markAsSeen(_error, {}, _description);
```

Comment:
I think you may have left the wrong line here :)
```suggestion
		m_errorReporter.warning(_error, _description);
```

Reply:
ah damn true